README.md

```diff
@@ -9,7 +9,7 @@
 [![npm version](https://badge.fury.io/js/igniteui-webcomponents.svg)](https://badge.fury.io/js/igniteui-webcomponents)
 [![Discord](https://img.shields.io/discord/836634487483269200?logo=discord&logoColor=ffffff)](https://discord.gg/39MjrTRqds)
 
-[Ignite UI for Web Components] is a complete library of UI components, giving you the ability to build modern web applications using encapsulation and the concept of reusable components in a dependency-free approach. All components are based on the [Indigo.Design Design System] and are backed by ready-to-use UI kit for Figma.
+[Ignite UI for Web Components] is a comprehensive library that includes the fastest Data Grid on the market, a high-performing Hierarchical Grid, Pivot Grid, 60+ data charts, Dock Manager, and more. Plus maps, gauges and other reusable feature-rich components to help you create better web apps and modern-day UX experiences.
```

Comment:
add this link https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/data-grid to Data Grid

add this link https://www.infragistics.com/webcomponentssite/components/grids/hierarchical-grid/overview to Hierarchical Grid

add this https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/charts/chart-overview to data charts

add this https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/layouts/dock-manager to Dock Manager